appeals/web/src/server/lib/mappers/data/appellant-case/submappers/split.cjs

```diff
@@ -0,0 +1,51 @@
+const fs = require('fs');
```

Comment:
This is an impressive script, but should it be removed now the refactoring is done? Or might it be needed in future?

Reply:
Yep there's a temporary file I've committed too 😅 got lost in the sauce